mkdocs.yml

```diff
@@ -53,6 +53,8 @@ nav:
         - Circuits: 'core-functionality/circuits.md'
         - Power Tracking: 'core-functionality/power.md'
         - Tenancy: 'core-functionality/tenancy.md'
+    - Docker:
+        - Overview: 'docker/index.md'
```

Comment:
This feels like it should be either inside the `Installation` section or at least adjacent to it. I'd also expect to see some sort of added content *in* the `Installation` docs pointing the reader towards this option as an alternative to the virtualenv-based installation workflow?